src/Conversion/ONNXToKrnl/Math/Elementwise.cpp

```diff
@@ -473,6 +473,33 @@ Value emitScalarOpFor<ONNXCastOp>(ConversionPatternRewriter &rewriter,
   return create.math.cast(elementType, scalarOperands[0]);
 }
 
+//===----------------------------------------------------------------------===//
```

Comment:
@hamptonm1 I thought we would avoid having CastLike because they are not optimized away in constant propagation. Isn't it too late to remove it here, during ONNX to Krnl lowering? I thought we would nuke them in Rewrite.